### PR TITLE
Add figma storybook integration

### LINKED ISSUE #143: Figma Storybook Add-on
We'd like to get Figma's [Storybook Add-on](https://www.figma.com/resources/extensions-and-apis/storybook-figma-addon/) integrated.

### PR DESCRIPTION
**Pull request by M0nica**
[Project Reno Jira Ticket](http://jira.nypl.org/browse/RENO-XXX)

[ResearchNow Jira Ticket](http://jira.nypl.org/browse/SFR-XXX)

## **This PR does the following:**
-

Add Figma Storybook integration and link to two Figma files for buttons 
Related to https://github.com/NYPL/nypl-design-system/issues/143:
 
Screen Shot 2020-05-28 at 12 46 35 PM
Screen Shot 2020-05-28 at 12 46 46 PM

### Front End Review:
- [ ] View [the example in Storybook](https://reno-XXX-nypl.pantheonsite.io/themes/custom/nypl_emulsify/pattern-lab/public)
- [ ] Check against the [Metronome documentation](http://themetronome.co/components/xxx/?fresh=true)

### CODE REVIEW

File: src/react-components/src/components/Button/Button.stories.tsx [hunk 1]
```diff
@@ -111,3 +113,25 @@ export const linkLikeButtonIconLeft = () => (
         <span>button content</span>
     </Button>
 );
+
+buttonFilledIcon.story = {
```
Comment (helenvholmes) [inline src/react-components/src/components/Button/Button.stories.tsx]: Can we maybe keep these alongside the story they add documentation for? i.e.

```
export const buttonFilledIcon = () => (
...
);

buttonFilledIcon.story = {
...
};
```

**Review by helenvholmes — CHANGES_REQUESTED** [review]
Small request on which lines the added code is in the story files, but mostly 🤩 this is all v v cool.

**Comment by hokei** [comment]
🎉 🎉

File: src/react-components/src/components/Button/Button.stories.tsx (hunk 1 above)
Comment (M0nica) [reply src/react-components/src/components/Button/Button.stories.tsx]: I updated to move the Figma links closer to their related stories!

**Review by helenvholmes — APPROVED** [review]
🎉 thank you thank you this looks excellent

**Pull request merged**